The Asset Manager is now visible (#330)
akeneo/pim-api-docs: content/concepts/target-market-settings.md

# Target market settings

> Product information enriched in the PIM is meant to be distributed over your channels. Otherwise gathering so many information into one single source of truth would be quite useless, right? :wink:

Reviewer: ```suggestion
Product information enriched in the PIM is meant to be distributed over your channels. Otherwise gathering so much information into one single source of truth would be quite useless, right? :wink:
```

Target market settings are here to specify those distribution channels. You can interact with these entities through the following resources.  

> Each section below contains an explanation of the concept behind these resources, what is its usage in the PIM and its JSON format in order to interact with the API. 

Reviewer: ```suggestion
Each section below contains an explanation of the concept behind these resources. You will find out more about their usage in the PIM and their JSON format in order for them to interact with the API. 
```
